app/controllers/provider_interface/conditions_controller.rb

```diff
@@ -1,66 +0,0 @@
-module ProviderInterface
-  class ConditionsController < ProviderInterfaceController
```

Comment:
Should we remove/redirect the old routes?

Reply:
I removed the routes and added a temporary redirect for conditions/edit that we can remove in a month or so.